Gcpe.Hub.API/Helpers/MappingProfile.cs

```diff
@@ -11,9 +11,7 @@ public MappingProfile()
         {
             CreateMap<Activity, Models.Activity>()
                 .ForMember(dest => dest.MinistriesSharedWith, opt => opt.MapFrom(src => src.ActivitySharedWith.Select(sw => sw.Ministry.Key)))
-                .ForMember(dest => dest.Categories, opt => opt.MapFrom(src => src.ActivityCategories.Select(ac => ac.Category.Name)))
-                .ForMember(dest => dest.StartDateTime, opt => opt.MapFrom(src => DateTime.SpecifyKind((DateTime)src.StartDateTime, DateTimeKind.Local)))
```

Comment:
why aren't these datetime mappings needed anymore? I was looking at these when I was trying to debug the tests so am curious

Reply:
nevermind i found the answer below